release.nix

```diff
@@ -6,65 +6,48 @@
 # them to all supported build platforms.
 #
 ############################################################################
-
 # The project sources
-{ cardano-prelude ? { outPath = ./.; rev = "abcdef"; }
-
-# Function arguments to pass to the project
-, projectArgs ? {
-    config = { allowUnfree = false; inHydra = true; };
-    gitrev = cardano-prelude.rev;
+{
```

Comment:
should we just bin this if we're switching to cicero or do we want to keep both for now?

Reply:
I think until we get mac agents for cicero we might want to keep both.